docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -89,6 +92,31 @@
           helm_chart_overrides: "/home/{{ ansible_ssh_user }}/{{ helm_chart_overrides | basename }}"
         when: helm_chart_overrides is defined
 
+      - block:
+
+        ## user not uploaded the artifacts
+        ## check in subcloud /usr/local/share/applications/
+
+        - set_fact:
+            helm_chart_overrides: "/usr/local/share/applications/overrides/wind-river-cloud-platform-deployment-manager-overrides-subcloud.yaml"
+            manager_chart: "/usr/local/share/applications/helm/wind-river-cloud-platform-deployment-manager-*.tgz"
+          
+        - name: Check if helm-chart file exists in subcloud
+          stat:
+            path: manager_chart
+          register: helm_chart_file
+         
+        - name: Check if helm-overrides file exists in subcloud
+          stat:
+            path: helm_chart_overrides
+          register: helm_chart_overrides_file
+          
+        - fail: Failing ansible-playbook when helm-chart and helm-chart overrides not exists in subcloud
```

Comment:
1 Normally, we would add a name and add the fail module under the task name. 
2 Please don't make the tasks name too long, this can be Fail when helm chart and overrides don't exist

Reply:
resolved.